### PR TITLE
Configs refactoring

### LINKED ISSUE #1622: Refactor config setup
Some configs are not used (ropsten, polygon etc), so we could simplify the setup, remappings etc...

### LINKED ISSUE #1624: Old licences
https://github.com/morpho-dao/morpho-v1/blob/41e4fae230ceff78f3284bc77c4748820e05b1b9/src/aave-v2/ExitPositionsManager.sol#L1

### PR DESCRIPTION
**Pull request by MathisGD**
Fixes #1622 and #1565

### CODE REVIEW

File: src/aave-v2/MorphoStorage.sol [hunk 1]
```diff
@@ -69,7 +69,7 @@ abstract contract MorphoStorage is OwnableUpgradeable, ReentrancyGuardUpgradeabl
     IEntryPositionsManager public entryPositionsManager;
     IExitPositionsManager public exitPositionsManager;
     IInterestRatesManager public interestRatesManager;
-    address public incentivesVault; // Deprecated.
+    address internal incentivesVault; // Deprecated.
```
Comment (MerlinEgalite) [inline src/aave-v2/MorphoStorage.sol:72]: I don't think we should do this because it's changing the ABI

**Review by MerlinEgalite — CHANGES_REQUESTED** [review]

File: src/aave-v2/MorphoStorage.sol (hunk 1 above)
Comment (MathisGD) [reply src/aave-v2/MorphoStorage.sol:72]: Yes I agree, I will remove the commit

**Comment by Rubilmax** [comment]
I think changes are noisy because it involves some diffs between the base branch and the head branch (for example: IncentivesVault removal). Can you update branches to make it clearer pls?

**Comment by MathisGD** [comment]
You are right. I will clean it. Passing this as draft for now.

**Comment by MathisGD** [comment]
Closing this as it is a lot of changes for not that much improvement.

**Pull request closed without merging**